paasta_tools/kubernetes/bin/kubernetes_remove_evicted_pods.py

```diff
@@ -131,20 +132,25 @@ def remove_pods(
 def evicted_pods_per_service(
     client: KubeClient,
 ) -> Mapping[str, Sequence[EvictedPod]]:
-    all_pods = get_all_pods(kube_client=client, namespace="")
-    evicted_pods = get_evicted_pods(all_pods)
-    log.info(f"Pods in evicted state: {[pod.metadata.name for pod in evicted_pods]}")
     evicted_pods_aggregated: Dict[str, List[EvictedPod]] = defaultdict(list)
-    for pod in evicted_pods:
-        service = get_pod_service(pod)
-        if service:
-            evicted_pods_aggregated[service].append(
-                EvictedPod(
-                    pod.metadata.name, pod.metadata.namespace, pod.status.message
+
+    for namespace in get_all_namespaces(client):
```

Comment:
should this be `get_all_managed_namespaces`

Reply:
possibly. I stuck with get_all_namespaces because I wanted to do a 1:1 port for now (and this only removes pods with the `paasta.yelp.com/service` label anyway, afaict).